MAINTAINERS.md

```diff
@@ -1,147 +1,17 @@
 - [Overview](#overview)
 - [Current Maintainers](#current-maintainers)
-- [Maintainer Responsibilities](#maintainer-responsibilities)
-  - [Uphold Code of Conduct](#uphold-code-of-conduct)
-  - [Prioritize Security](#prioritize-security)
-  - [Review Pull Requests](#review-pull-requests)
-  - [Triage Open Issues](#triage-open-issues)
-  - [Backports](#backports)
-  - [Be Responsive](#be-responsive)
-  - [Maintain Overall Health of the Repo](#maintain-overall-health-of-the-repo)
-  - [Manage Roadmap](#manage-roadmap)
-  - [Add Continuous Integration Checks](#add-continuous-integration-checks)
-  - [Use Semver](#use-semver)
-  - [Release Frequently](#release-frequently)
-  - [Promote Other Maintainers](#promote-other-maintainers)
-  - [Describe the Repo](#describe-the-repo)
-- [Becoming a Maintainer](#becoming-a-maintainer)
-  - [Nomination](#nomination)
-  - [Interest](#interest)
-  - [Addition](#addition)
-- [Removing a Maintainer](#removing-a-maintainer)
-  - [Moving On](#moving-on)
-  - [Inactivity](#inactivity)
-  - [Negative Impact on the Project](#negative-impact-on-the-project)
 ## Overview
 
-This document explains who the maintainers are (see below), what they do in this repo, and how they should be doing it. If you're interested in contributing, see [CONTRIBUTING](CONTRIBUTING.md).
+[This document](https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md#maintainer-responsibilities) explains who the maintainers are (see below), what they do in this repo, and how they should be doing it. If you're interested in contributing, see [CONTRIBUTING](CONTRIBUTING.md).
```

Comment:
Saying "This document" linking to another document, while explaining the contents of the current document and the linked one is somewhat confusing. I think it'd be better to rephrase this to something like `This document lists the current maintainers of this repository. An explanation of what they do and their responsibilities can be found in the [central opensearch project MAINTAINERS file]`

Reply:
I hit merge - this is a valid comment though, we have this language all over the place. Something to add to a template ...